jerry-core/ecma/operations/ecma-regexp-object.c

```diff
@@ -2499,10 +2499,13 @@ ecma_regexp_split_helper (ecma_value_t this_arg, /**< this value */
  */
 static ecma_value_t
 ecma_regexp_replace_helper_fast (ecma_replace_context_t *ctx_p, /**<replace context */
-                                 const re_compiled_code_t *bc_p, /**< regexp bytecode */
+                                 ecma_extended_object_t *re_obj_p, /**< regexp object */
                                  ecma_string_t *string_p, /**< source string */
                                  ecma_value_t replace_arg) /**< replace argument */
 {
+  const re_compiled_code_t *bc_p = ECMA_GET_INTERNAL_VALUE_ANY_POINTER (re_compiled_code_t,
```

Comment:
Since we assert that `bc_p` is not null below, how about using  `ECMA_GET_INTERNAL_VALUE_POINTER` instead?